### ISSUE TITLE
Ember 3.15 - Modifier test always imports hbs from htmlbars-inline-precompile

### ISSUE THREAD

**Issue by ijlee2** [ember-modifier/ember-modifier#10]
## Description

The blueprint for a modifier test hardcoded the hbs import statement.

```javascript
import hbs from 'htmlbars-inline-precompile';
```

https://github.com/ember-modifier/ember-modifier/blob/master/blueprints/modifier-test/qunit-files/__root__/integration/__collection__/__name__-test.js#L4

For Ember 3.15, which uses `ember-cli-htmlbars ^4.2.0`, I believe the blueprint would need to generate the line:
```javascript
import { hbs } from 'ember-cli-htmlbars';
```

https://github.com/ember-cli/ember-cli-htmlbars#tagged-template-usage--migrating-from-htmlbars-inline-precompile

## Proposed Solution

The blueprint for a component test shows how we can check the version of `ember-cli-htmlbars` in order to dynamically add the import statement:

https://github.com/emberjs/ember.js/blob/master/blueprints/component-test/index.js#L89-L97

https://github.com/emberjs/ember.js/blob/master/blueprints/component-test/qunit-rfc-232-files/__root__/__testType__/__path__/__test__.js#L4

Please check both QUnit and Mocha.

**Comment by rwjblue** [comment]
Thanks for reporting! The proposed solution seems good, would definitely love a pull request fixing this!

**Comment by rwjblue** [comment]
Fixed by #46
